mock_service/lib/dsl.py

```diff
@@ -72,9 +75,12 @@ def get_datapoint_value(context: ExecutionContext, path: str, default: Any = 0)
     """
     if path not in _mocked_datapoints:
         _required_datapoint_paths.append(path)
-
-    if path in context.datapoints_cache:
-        return context.datapoints_cache[path].value
+    context.client.connect()
```

Comment:
Not sure if connecting and disconnecting is better than caching the values. Was this change driven by a bug?

Reply:
yeah not sure too about this, but seemed like a better option at the point I made the change. For me it would be now okay to revert to caching if you think it is faster/better.

Reply:
Without numbers its hard to say which one is better. You are trading runtime performance vs memory usage. Biggest downside I see is the number of log messages you are generating by connecting/disconnecting. Can the client somehow stay connected?

Reply:
Yes we can give over the client from baseservice then we have just on client that is connected at the beginning and then serves for all calls made to databroker. Will change it to this